dev/bin/knative/resources/knative-eventing.yaml

```diff
@@ -0,0 +1,4708 @@
+---
```

Comment:
Please tell me this is copy and pasted from somewhere!

Whilst understanding what it is I wouldn't like to write it from scratch.

Reply:
@manstis this is the installation file provided by @matzew